diff -ur ledit-1.10/pa_local.ml ledit-1.10.x/pa_local.ml
--- ledit-1.10/pa_local.ml	2001-06-16 10:50:02.000000000 +0200
+++ ledit-1.10.x/pa_local.ml	2006-04-21 16:12:49.000000000 +0200
@@ -14,10 +14,10 @@
 open Pcaml;
 
 value expr_of_patt p =
-  let loc = MLast.loc_of_patt p in
+  let _loc = MLast.loc_of_patt p in
   match p with
   [ <:patt< $lid:x$ >> -> <:expr< $lid:x$ >>
-  | _ -> Stdpp.raise_with_loc loc (Stream.Error "identifier expected") ]
+  | _ -> Stdpp.raise_with_loc _loc (Stream.Error "identifier expected") ]
 ;
 
 EXTEND
