Common subdirectories: ledit-1.10.orig/CVS and ledit-1.10/CVS
diff -u ledit-1.10.orig/Makefile ledit-1.10/Makefile
--- ledit-1.10.orig/Makefile	Sat Jun 16 15:28:43 2001
+++ ledit-1.10/Makefile	Mon May 13 00:33:09 2002
@@ -19,7 +19,7 @@
 	sed s/LEDIT_VERSION/$$VERSION/ ledit.l.tpl > ledit.l
 
 pa_local.cmo: pa_local.ml
-	$(PP) pa_extend.cmo q_MLast.cmo -warn_seq pa_local.ml -o pa_local.ppo
+	$(PP) pa_extend.cmo q_MLast.cmo pa_local.ml -o pa_local.ppo
 	$(COMP) -I `camlp4 -where` -c -impl pa_local.ppo
 	/bin/rm -f pa_local.ppo
 
@@ -42,7 +42,7 @@
 include .depend
 
 .ml.cmo:
-	$(PP) ./pa_local.cmo -warn_seq $< -o $*.ppo
+	$(PP) ./pa_local.cmo $< -o $*.ppo
 	$(COMP) -I `camlp4 -where` -c -impl $*.ppo
 	/bin/rm -f $*.ppo
 .mli.cmi:
Only in ledit-1.10: Makefile~
